modules/browsemap/src/test/java/com/sygic/module/browsemap/BrowseMapFragmentViewModelTest.kt

```diff
@@ -100,12 +100,12 @@ class BrowseMapFragmentViewModelTest {
         whenever(locationManager.positionOnMapEnabled).thenReturn(true)
 
         val arguments = mock<Bundle>()
-        whenever(arguments.getInt(eq(KEY_MAP_SELECTION_MODE), any<Int>())).thenReturn(MapSelectionMode.FULL)
-        whenever(arguments.getBoolean(eq(KEY_POSITION_ON_MAP), any<Boolean>())).thenReturn(true)
-        whenever(arguments.getBoolean(eq(KEY_COMPASS_ENABLED), any<Boolean>())).thenReturn(true)
-        whenever(arguments.getBoolean(eq(KEY_COMPASS_HIDE_IF_NORTH), any<Boolean>())).thenReturn(true)
-        whenever(arguments.getBoolean(eq(KEY_POSITION_LOCK_FAB), any<Boolean>())).thenReturn(true)
-        whenever(arguments.getBoolean(eq(KEY_ZOOM_CONTROLS), any<Boolean>())).thenReturn(true)
+        whenever(arguments.getInt(eq(KEY_MAP_SELECTION_MODE), any())).thenReturn(MapSelectionMode.FULL)
```

Comment:
nie `any<>`?

Reply:
? :) ne-e :)

Reply:
no ja nemyslim toto https://static.javadoc.io/org.mockito/mockito-core/1.9.5/org/mockito/Matchers.html#any(), ale to od nhaarman

Reply:
aj ked pozera, ze on tam ma refied typ, ale nie je to genericka funkcia... a vysledok je na zaklade `Mockito.any(Class)`. Tak ok, ked nechce generika tak staci tak